core/db/migrate/20161029130019_add_address_format_settings_to_countries.rb

```diff
@@ -0,0 +1,9 @@
+class AddAddressFormatSettingsToCountries < ActiveRecord::Migration[5.0]
+  def change
+    add_column :spree_countries, :provence_label, :string
```

Comment:
I would also prefer we keep as much data in Carmen (or even just hardcoded) as possible.